docs/bgp.md

```diff
@@ -144,9 +144,38 @@ kubectl annotate node <kube-node> "kube-router.io/peer.asns=65000,65000"
 kubectl annotate node <kube-node> "kube-router.io/peer.passwords=U2VjdXJlUGFzc3dvcmQK,"
 ```
 
+Finally, to include peer passwords as a file you would run kube-router with the following option:
+```
+--peer-router-ips="192.168.1.99,192.168.1.100"
+--peer-router-asns="65000,65000"
+--peer-router-passwords-file='/etc/kube-router/bgp-passwords.conf'
```

Comment:
any reason this would require single quotes?

Reply:
Fixed! Thanks.